### PR TITLE
Composer update 

### PR DESCRIPTION
**Pull request by HLeithner**
Update composer

- psr/log (definition only)
- joomla/filesystem (php 7.4 fixes)
- joomla/event (mainly cleanups)

Code Review needed

### CODE REVIEW

File: libraries/vendor/joomla/filesystem/meta/language/en-GB/en-GB.lib_joomla_filesystem_patcher.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.
-; License GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
```
Comment (brianteeman) [inline libraries/vendor/joomla/filesystem/meta/language/en-GB/en-GB.lib_joomla_filesystem_patcher.ini]: This change is not correct  - is it something I need to patch upstream?

Comment (HLeithner) [reply libraries/vendor/joomla/filesystem/meta/language/en-GB/en-GB.lib_joomla_filesystem_patcher.ini]: yes

**Review by wilsonge — APPROVED** [review]

File: libraries/vendor/joomla/filesystem/meta/language/en-GB/en-GB.lib_joomla_filesystem_patcher.ini (hunk 1 above)
Comment (brianteeman) [reply libraries/vendor/joomla/filesystem/meta/language/en-GB/en-GB.lib_joomla_filesystem_patcher.ini]: and done

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on deeece9d6811ab107008ab3a7c19992ea5b65d81<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26879">issues.joomla.org/tracker/joomla-cms/26879</a>.</sub>

**Pull request merged**